jenkins/bokchoy_db_pull_request.py

```diff
@@ -0,0 +1,212 @@
+import sys
```

Comment:
Could you please put in some high level documentation explaining the purpose of the script, especially with regard to the interaction with paver/s3? This will be useful for new hires, people outside of our team